docs/user/commandline_old_commands.rst

```diff
@@ -0,0 +1,72 @@
+Old Command Line Commands
+=========================
+
+In GeoMesa 1.3.0 some command names have changed. Please reference the chart below for more information.
+
+Accumulo Command Line Tools
+---------------------------
+
++---------------------+--------------------+
+| Old                 | New                |
++=====================+====================+
+| add-attribute-index	| add-attributeindex |
```

Comment:
Alas, Sphinx doesn't seem to like the mix of spaces and tabs in this table. I get an error about a malformed table when I try to build it.